DataFormats/Reconstruction/include/ReconstructionDataFormats/TrackFwd.h

```diff
@@ -126,11 +127,11 @@ class TrackParCovFwd : public TrackParFwd
   TrackParCovFwd() = default;
   ~TrackParCovFwd() = default;
   TrackParCovFwd& operator=(const TrackParCovFwd& tpf) = default;
-  TrackParCovFwd(const Double_t z, const SMatrix5 parameters, const SMatrix55 covariances, const Double_t chi2);
+  TrackParCovFwd(const Double_t z, const SMatrix5 parameters, const SMatrix55Sym covariances, const Double_t chi2);
```

Comment:
I guess here one should pass the matrices by ref. not by copy, i.e. `TrackParCovFwd(const Double_t z, const SMatrix5& parameters, const SMatrix55Sym& covariances, const Double_t chi2)` ?

Reply:
Done